clonedigger.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+clonedigger pylast
+! grep -E "Clones detected\|lines are duplicates" output.html
```

Comment:
Ignoring the ! for now, this doesn't work when clones are detected (and there are right now). A simple test on Travis:
```
$ echo "Clones detected" | grep -E "Clones detected\|lines are duplicates"
$ echo "Clones detected" | grep -E "Clones detected|lines are duplicates"
Clones detected
$ echo "Clones detected" | grep -e "Clones detected\|lines are duplicates"
Clones detected
$ echo "Clones detected" | grep -e "Clones detected|lines are duplicates"
```